modules/tinymce/src/plugins/image/main/ts/ui/DialogInfo.ts

```diff
@@ -35,7 +35,7 @@ const collect = (editor: Editor): Promise<ImageDialogInfo> => {
   const hasAdvTab = Settings.hasAdvTab(editor);
   const hasUploadTab = Settings.hasUploadTab(editor);
   const hasUploadUrl = Settings.hasUploadUrl(editor);
-  const hasUploadHandler = Settings.hasUploadHandler(editor);
+  // const hasUploadHandler = Settings.hasUploadHandler(editor);
```

Comment:
Please remove this, rather than leave this commented out.

Reply:
Ops... :-)

Reply:
Done